support-frontend/assets/components/headers/header/header.scss

```diff
@@ -213,3 +219,103 @@ $header-links-min-width: gu-span(12);
     }
   }
 }
+
+.component-header--display-checkout {
+  .component-header__wrapper {
+    max-width: 50rem;
+
+    @include mq($from: tablet, $until: desktop) {
+      max-width: 100%;
+      padding: 0;
+      margin: 0;
+    }
+
+    @include mq($from: leftCol) {
+      max-width: 60rem;
+    }
+
+    .component-header-topnav {
+      @include mq($until: tablet) {
+        justify-content: space-between;
+      }
+
+      @include mq($from: tablet, $until: desktop) {
+        justify-content: flex-end;
+      }
+
+      .component-header-topnav__utility {
+        display: none;
+      }
+
+      .component-header-topnav__checkout {
+        display: flex;
+        flex: 0 0 490px;
```

Comment:
try to use `gu-span(*)` here rather than a fixed pixel size

Reply:
The issue here is that the lines in the nav need to line up with the structure of the main body and I don't seem to be able to get it right with whole columns. Am I going about this the wrong way?